docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -149,9 +149,11 @@ sudo apt install ziti-edge-tunnel
 
   ```bash
   sudo -u ziti tee /opt/openziti/etc/identities/ziti-id.jwt >/dev/null
-  # paste the contents of the enrollment token JWT file and press Ctrl+D
+  # run the above command then paste the token from clipboard then press Ctrl+D to close the file
```

Comment:
seems like we should just tell the user to put it somewhere of their choosing. also the whole ctrl+d thing, i get why you're doing it but i'd prefer to keep it all simpler... download the file... use it...